api/src/envVarsSchema.ts

```diff
@@ -77,7 +77,7 @@ export const envVarsSchema = Joi.object({
     ),
   JWT_ALGORITHM: Joi.string()
     .default("HS256")
-    .valid("HS256", "RS256")
+    .allow("HS256", "RS256", "")
```

Comment:
When I tested with `JWT_ALGORITHM=` in .env file, it didn't default to "HS256".
Apprently to allow "" and use the dafult value, more changes are required.

This worked for me:

JWT_ALGORITHM: Joi.string()
    .allow("HS256", "RS256", "", null)
    .empty(["", null])
    .default("HS256")
    .note("Algorithm used for signing and verifying JWTs."),